gridt/src/app/home/home.page.ts

```diff
@@ -190,7 +192,12 @@ export class HomePage implements OnInit, OnDestroy {
         {
           text: "Yes",
           handler: (data) => {
-            this.signal(movement, data.message);
+            if(data.message !==null && data.message.length < 140){
```

Comment:
Could you set spaces around the if statements? It reads a little cramped. See line 162 for an example.